.circleci/config.yml

```diff
@@ -49,6 +49,12 @@ jobs:
       - run: docker run --privileged linuxkit/binfmt:af88a591f9cc896a52ce596b9cf7ca26a061ef97
       - run: promu crossbuild -v --parallelism $CIRCLE_NODE_TOTAL --parallelism-thread $CIRCLE_NODE_INDEX
       - run: promu --config .promu-cgo.yml crossbuild -v --parallelism $CIRCLE_NODE_TOTAL --parallelism-thread $CIRCLE_NODE_INDEX
+      # sign the darwin build so it doesn't get SIGKILLed on start, see: https://github.com/prometheus/node_exporter/issues/2539
+      - run:
+          command: |
+            if [ -d "$(pwd)/.build/darwin-arm64" ]; then
```

Comment:
Should we also do this for amd64?

Also, let's be explicit about checking for the file.

```suggestion
            if [[ -f "$(pwd)/.build/darwin-arm64/node_exporter" ]]; then
```

Reply:
Checking for the file now.

Good idea to also sign amd64, I'll add another if statement for it and test how it goes. But, I don't have amd64 mac to test the artifact with.